src/app/backend/resource/pod/poddetail.go

```diff
@@ -15,17 +15,23 @@
 package pod
 
 import (
+	"encoding/json"
 	"log"
 
 	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/apis/batch"
 	k8sClient "k8s.io/kubernetes/pkg/client/unversioned"
 
 	"github.com/kubernetes/dashboard/src/app/backend/client"
 	"github.com/kubernetes/dashboard/src/app/backend/resource/common"
 	"github.com/kubernetes/dashboard/src/app/backend/resource/dataselect"
+	"github.com/kubernetes/dashboard/src/app/backend/resource/job/joblist"
 	"github.com/kubernetes/dashboard/src/app/backend/resource/metric"
 )
 
+const (
+	CreatedByAnnotation = "kubernetes.io/created-by"
```

Comment:
Maybe we spawn an issue and add a todo here to update the deps, so that Rob is unblocked?